sw/device/exts/common/flash_crt.S

```diff
@@ -2,13 +2,22 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
+/**
+ * Flash header.
+ *
+ * Contains the address of the ENTRY point. This is harcoded to the _start
+ * symbol for now.
```

Comment:
Now we've moved the header to this file, I wouldn't say the entry point was hard-coded, as the `_start` symbol is defined below.

Reply:
Nit: Also, "har**d**coded"

Reply:
I was unsure about that, since it's true that we still aren't automatically updating the `flash_header` pointer if we change the entry point symbol, but I have no problem with that point of view, so I'll update it.